libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccount.m

```diff
@@ -141,25 +142,33 @@ - (NSString *)photoPathInternal:(NSError**)error {
     return [userPhotoPath stringByAppendingPathComponent:[SFDirectoryManager safeStringForDiskRepresentation:_credentials.userId]];
 }
 
-- (UIImage*)photo {
+- (UIImage *)photo {
     if (nil == _photo) {
```

Comment:
@bhariharan I realized this has a check so it only goes to disk when nil so I didn't move it to a fetch method like we talked about before